utils/kata-manager.sh

```diff
@@ -720,26 +721,74 @@ handle_containerd()
 	containerd --version
 }
 
+handle_docker()
+{
+	{ containerd_installed; ret=$?; } || true
+	if [ "$ret" -eq 0 ]
+	then
+		info "Backing up previous containerd configuratinn"
+		local cfg="/etc/containerd/config.toml"
+
+		sudo mv $cfg $cfg.system-$(date -Iseconds)
+	fi
+
+	containerd_installed
+
+	local tmpdir
+	tmpdir=$(mktemp -d)
+	
```

Comment:
Can you delete these 3 lines as they cause the following error (since `tmpdir` is already defined globally):

```
utils/kata-manager.sh: line 737: local: tmpdir: readonly variable
```